src/main/java/com/google/flourbot/api/Bot.java

```diff
@@ -26,20 +26,27 @@
 import java.security.GeneralSecurityException;
 import java.util.logging.Logger;
 import java.io.FileNotFoundException;
+import java.util.Map;
+import java.util.HashMap;
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.util.Collections;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.LinkedList;
 
 @SpringBootApplication
 @RestController
 public class Bot {
+
   static final String CHAT_SCOPE = "https://www.googleapis.com/auth/chat.bot";
   private static final String SERVICE_ACCOUNT = "/service-acct.json";
   private static final Logger logger = Logger.getLogger(Bot.class.getName());
+  private static String helpMessage;
```

Comment:
Add the constant definition here

Reply:
done!

Reply:
Like this one. Oh I see, if there's 2 different messages for DM or a room, should we have 2 constants? 
`"To use a room member's macro, the creator of a macro must write \"@MacroBot initiate MacroName \". " + 
            "Once this has been sent, any room member can use the initialized macro by writing \"@MacroBot MacroName <your message> \" " + 
            "If your macro has already been used in a thread, you may omit the MacroName and can simply write \"@MacroBot <your message>\".";`